### ISSUE TITLE
[Accessibility] NVDA and JAWS do not read focused auto-suggestion item properly in terminal input

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#131090]
CC @isidorn 

Does this issue occur when all extensions are disabled?: Yes

## System Info

Version: 1.60.0-insider (system setup)
Commit: 0214d59846f265d9e50f53b61e2267407d81cbfb
Date: 2021-08-18T05:14:14.284Z
Electron: 13.1.8
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.19043

Steps to Reproduce:

1. Install [R for Windows](https://cran.r-project.org/bin/windows/).
2. Add the R path to system path variable. For example, "C:/Program Files/R/R-4.1.1/bin/x64"
3. Install [radian](https://github.com/randy3k/radian).
4. Open VSCode with [NVDA](https://github.com/nvaccess/nvda).
5. Open terminal.
6. Type `radian` to start radian.
7. In radian session, type `lib` as in "library" and arrow-down.
8. See if NVDA reads properly any focused auto-suggestion.

* Note: Currently, NVDA and JAWS do not read focused auto-suggestion item properly in terminal input. I imagine this issue would also happen in bash shell auto-suggestion/completion.

**Comment by isidorn** [comment]
@jooyoungseo demonstrated this issue to me, the problem seem to be that in this case a custom suggest box appears in the terminal. And this suggest box does not get properly read. I am not sure if we can control this custom suggest box, or how we would fix this.

**Comment by jooyoungseo** [comment]
@leonardder Do you think this issue has to do with NVDA? Can this be handled on NVDA's end?

**Comment by jooyoungseo** [comment]
@leonardder, it seems like this issue happens in CMD.exe as well.

**Comment by isidorn** [comment]
@jooyoungseo yeah, I think this is a custom element, and VS Code I think does not have control over how it is being rendered.
Maybe the best would be file an issue against radian https://github.com/randy3k/radian

**Comment by leonardder** [comment]
How does VS Code read the selection when arrowing through something interactive in Bash? I can't come up with a suggestion that quick.
I assume the selected item is highlighted, which triggers a redraw of several suggestions that causes VS Code to think that there's new content in the console.

**Comment by isidorn** [comment]
I am not sure how this custom rendered suggest boxes work in the terminal. Maybe @Tyriar knows, and if not I can give it a try and investigate how this works.

**Comment by Tyriar** [comment]
We have no control over how auto complete is presented in the terminal, we just get a string of characters and cursor instructions to render to the screen. Not sure why this isn't working.
